front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/HeaderImageDropzone.tsx

```diff
@@ -36,7 +36,7 @@ interface Props {
   onAdd: (newImage: UploadFile[]) => void;
   onRemove: () => void;
   headerError: string | null;
-  header_bg: UploadFile[] | null;
```

Comment:
Didn't understand why this needs to be an array if we can only add 1 file at most.